modules/ossm-security-cert-manage.adoc

```diff
@@ -44,103 +44,63 @@ spec:
       istiod:
         type: PrivateKey
         privateKey:
-          rootCADir:  /etc/cacerts
-----
-+
-. To make sure the workloads add the new certificates promptly, delete the secrets generated by {ProductShortName}, named `istio.*`. In this example, `istio.default`. {ProductShortName} issues new certificates for the workloads.
-+
-[source,terminal]
-----
-$ oc delete secret istio.default
+          rootCADir: /etc/cacerts
 ----
 
+[NOTE]
+====
+After creating/changing/deleting the `cacert` secret, control plane `istiod` and `gateway` pods need to be restarted so changes get in effect. You can use a command like this to trigger the restart: `oc -n istio-system delete pods -l 'app in (istiod,istio-ingressgateway)'`.
```

Comment:
@jwendell how about we rephrase "...control plane `istiod` and `gateway` pods need to be restarted so changes get in effect." to "...control plane restart the `istiod` and `gateway` pods so the changes go into effect."

For "You can use a command like this to trigger the restart:" can we say "Use the following command to restart the pods:"